src/operator/elemwise_op_common.h

```diff
@@ -156,6 +157,75 @@ inline bool ElemwiseStorageType(const nnvm::NodeAttrs& attrs,
     attrs, in_attrs, out_attrs);
 }
 
+template<int n_out>
```

Comment:
Can this function be removed? Since we have the default storage type function which assigns kdefault storage.